tcms/rpc/api/forms/testrun.py

```diff
@@ -21,40 +23,51 @@ def assign_plan(self, plan_id):
         self.populate(self.fields['plan'].queryset.first().product_id)
 
 
-class UpdateForm(NewForm):
-    plan = forms.ModelChoiceField(
-        queryset=TestPlan.objects.all(),
-        required=False,
-    )
-    summary = forms.CharField(
-        required=False
-    )
-    manager = forms.ModelChoiceField(
-        queryset=User.objects.all(), required=False
-    )
-    product = forms.ModelChoiceField(
-        queryset=Product.objects.all(),
-        empty_label=None, required=False
-    )
-    product_version = forms.ModelChoiceField(
-        queryset=Version.objects.none(),
-        empty_label=None, required=False
-    )
-    build = forms.ModelChoiceField(
-        queryset=Build.objects.all(),
-        required=False
+class UpdateForm(forms.ModelForm):
+
+    manager = UserField()
+    default_tester = UserField(required=False)
+
+    start_date = forms.DateTimeField(
```

Comment:
This field isn't required. I'm not sure if the overridden constructor will take care of that. It should take care of it which means the `required` attribute isn't really needed anywhere in this class. Whatever you chose to do keep it consistent.

Reply:
do you mean that we don't need `start_date` or something else?

Reply:
I mean the `required` attribute.